pkg/cmd/secret/list/list.go

```diff
@@ -139,6 +139,10 @@ func listRun(opts *ListOptions) error {
 		return fmt.Errorf("failed to get secrets: %w", err)
 	}
 
+	if len(secrets) == 0 {
+		return utils.HandleNoResults(opts.IO, "No secrets found")
```

Comment:
Despite comments, this is working correctly for repos without any secrets.

Reply:
omg i just can't spell my username :|